vertexarray.cpp

```diff
@@ -4,8 +4,7 @@
 
 #include <vector>
 
-VertexArray::VertexArray() :
-    mRendererID(0)
+VertexArray::VertexArray()
```

Comment:
One more high level thing!! You should be using a namespace in almost all of your custom files.

So in sources and headers, wrap everything after your includes in a:

```
#include <something>

...

namespace pong
{

VertexArray::VertexArray()
...

}
```

Why is this useful? Well, remember at spatial where everything was prefixed by `spl`? the spirit of that idea was good, but stuck in 1989. The idea is that at some point you may want to differentiate between YOUR VertexArray, and some other implementation. For instance, if you bring in some TPS that also defines a VertexArray, you may be in trouble.

Having everything in a custom namespace is considered a good practice. It allows you to group like ideas together and avoid collisions.

Reply:
Interesting, yeah I will tackle that in a bit and going forward.